src/Form/FieldInput.js

```diff
@@ -86,7 +86,7 @@ class FieldInput extends React.Component {
 
     if (this.hasError()) {
       errorMsg = (
-        <p className={props.helpBlockClass}>
+        <p className={classNames(props.helpBlockClass)}>
```

Comment:
hmmm maybe what you want here is `classNames('default-class default-class-super-short', props.helpBlockClass)`? that way you have your defaults, and the props will either extend or override according to `classnames/dedupe`